test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml

```diff
@@ -4,7 +4,8 @@
 
 - set_fact:
     mode: delay
-  when: platform is not search('N5K|N6K|N7K') and ((imagetag != 'I2'))
+  when: platform is not search('N3K|N5K|N6K|N7K') and imagetag is not search('A8|I2') and
+    image_version is not search ('9.2') and chassis_type is not search('C9508')
```

Comment:
Why did you need the `chassis_type` here?

Reply:
Delayed LACP is not supported on Layer 3 port channels, FEX modules, or the Cisco Nexus 9500 Series switch. Platform for 9500 still set as N9K, I used chassis_type to actually check its a 9500 switch. And, I realized, I need to broaden the search to include other 9500 series switches C9504,C9508,C9516. 

I will do 
#######
chassis_type is not search('C95')
########

Reply:
Gotcha.  Can you add a comment in the test to indicate this?

Reply:
Done